src/tasks/simulate_uni_auction.ts

```diff
@@ -148,7 +157,39 @@ const simulateETHGNOAuction: () => void = () => {
     console.log("proposals executed")
     ////////////////////////////////////////////////////////////////////////////////
-    // 4nd: All kind of checks
+    // 4th: Check execution results
+    ////////////////////////////////////////////////////////////////////////////////
+    console.log('\n--------------------------------------------------------------------');
+    let gnosisDAOEthBalanceAfterWithdraw = await gnoToken.balanceOf(gnosisDAO.address)/1e18;
+    console.log(`DAO GNO Balance before the proposal started: ${initialGnoBalance/1e18}`);
+    console.log(`DAO GNO Balance after the proposal ended   : ${gnosisDAOEthBalanceAfterWithdraw}`);
+    let withdrawnTokensAfter = await vesting.withdrawnTokens();
+    console.log(`Withdrawn GNO Tokens from Vesting before Withdraw: ${initialWithdrawnTokens/1e18}`);
+    console.log(`Withdrawn GNO Tokens from Vesting after Withdraw : ${withdrawnTokensAfter/1e18}`);
+    let xdaiGovGNOBalanceAfter = await gnoToken.balanceOf(xdaiGovMultisigAddress);
+    console.log(`xDAI GNO Balance before transfer: ${initialxDaiGovGNOBalance/1e18}`);
+    console.log(`xDAI GNO Balance after transfer : ${xdaiGovGNOBalanceAfter/1e18}`);
+    let gnosisDAOwethBalanceAfter = await wethToken.balanceOf(gnosisDAO.address);
+    console.log(`DAO WETH balance before the proposal started: ${initialWethBalance/1e18}`);
+    console.log(`DAO WETH balance after the proposal ended   : ${gnosisDAOwethBalanceAfter/1e18}`);
+    let auctionWethBalanceAfter = await wethToken.balanceOf(easyAuction.address);
+    console.log(`Auction WETH balance before the proposal started: ${initialAuctionWethBalance/1e18}`);
+    console.log(`Auction WETH balance after the proposal ended   : ${auctionWethBalanceAfter/1e18}`);
+    let gnosisDAOETHBalanceAfter = await hardhatRuntime.ethers.provider.getBalance(gnosisDAO.address);
+    console.log(`DAO ETH Balance before the proposal started: ${initialETHBalance}`);
+    console.log(`DAO ETH Balance after the proposal ended   : ${gnosisDAOETHBalanceAfter}`);
+    console.log('--------------------------------------------------------------------\n');
+
+    // We expect the balance of xDAI Multisig to increase by the amount of GNO we withdraw from VestingContract
+    assert.ok(xdaiGovGNOBalanceAfter.eq(initialxDaiGovGNOBalance.add(gnoWithdrawAmount)));
+    // We expect 20,000 ETH to be changed for WETH and be deposited on the Auction contract
+    assert.ok(gnosisDAOwethBalanceAfter.eq(initialWethBalance));
+    assert.ok(auctionWethBalanceAfter.sub(initialAuctionWethBalance).eq(auctionedSellAmount));
+    assert.ok(gnosisDAOETHBalanceAfter.eq(initialETHBalance.sub(auctionedSellAmount)));
+
+
+    ////////////////////////////////////////////////////////////////////////////////
+    // 5th: All kind of checks
     ////////////////////////////////////////////////////////////////////////////////
     const GNOBalanceFromVestingClaim = (await gnoToken.balanceOf(gnosisDAO.address)).sub(initialGnoBalance)
```

Comment:
This test then makes no longer sense. It's covered by your assert statements. You can delete it